### PR TITLE
Validation eui 257

### PR DESCRIPTION
**Pull request by fazanki**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/org-manager/containers/org-pending-approve/org-pending-approve.component.html [hunk 1]
```diff
@@ -3,9 +3,21 @@
     <main class="govuk-main-wrapper" id="main-content" role="main">
         <div class="govuk-grid-row">
             <div class="govuk-grid-row">
+              <div *ngIf="true" id="errorSummary" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+                   data-module="error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-title">
+                  Sorry, there is a problem with the service.
```
Comment (fazanki) [inline src/org-manager/containers/org-pending-approve/org-pending-approve.component.html]: display message from store rather then hard coded msg

File: src/org-manager/store/reducers/org-pending.reducer.ts [hunk 2]
```diff
@@ -59,12 +62,21 @@ export function reducer(
       };
     }
 
+    case PendingOrgActionTypes.APPROVE_PENDING_ORGANISATIONS_FAIL: {
+      const responseMessages = action.payload;
```
Comment (fazanki) [inline src/org-manager/store/reducers/org-pending.reducer.ts]: dont put all the error message object returned by the server by maybe just a message and status code

File: src/org-manager/containers/org-pending-approve/org-pending-approve.component.ts [hunk 3]
```diff
@@ -27,6 +28,9 @@ export class OrgPendingApproveComponent implements OnInit, OnDestroy {
                     this.onGoBack();
                 }
             });
+        this.store.pipe( select(fromOrganisationPendingStore.pendingOrganisations)).subscribe(something => {
+           console.log(something['responseMessages']);
```
Comment (fazanki) [inline src/org-manager/containers/org-pending-approve/org-pending-approve.component.ts]: create selector that returns the error message from the state and hook it to html

**Comment by Vamshi-Muniganti** [comment]
no more valid as per Franco

**Pull request closed without merging**